### PR TITLE
[Improvement] Optimize selection prompt information when querying in yarn application mode; Added Run button to check select execute

### LINKED ISSUE #3237: [Improvement] Optimize selection prompt information when querying in yarn application mode
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar optimization requirement.

### Description

Currently, if you select application mode and click query, a stack error will appear, making it difficult to troubleshoot the problem. It needs to be optimized to an unsupported prompt message.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LeonYoah**
## Brief change log
#3237 
针对于select场景做了校验：
[查询]按钮不支持 yarn cluster模式和k8s cluster模式
dinky优化1

[运行]按钮执行时对select 语句的异常做了提示：
dinky优化2

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java [hunk 1]
```diff
@@ -335,6 +335,11 @@ public JobResult debugTask(TaskDTO task) throws Exception {
         task.setUseResult(true);
         // Debug mode need execute
         task.setStatementSet(false);
+        // mode check
+        GatewayType.get(task.getType());
```
Comment (Zzm0809) [inline dinky-admin/src/main/java/org/dinky/service/impl/TaskServiceImpl.java]: 此行有何用处么???

**Comment by Zzm0809** [comment]
需要补充一下 en 的国际化

**Comment by aiwenmo** [comment]
mvn spotless:apply

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Comment by Pandas886** [comment]
useful pr!

**Pull request merged**